json_schema/module/project/hca_bionetwork.json

```
@@ -0,0 +1,79 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "description": "Information about whether the project is part of a HCA Bionetwork or HCA Atlas.",
+    "additionalProperties": false,
+    "required": [
+    ],
+    "title": "HCA Bionetwork",
+    "name": "hca_bionetwork",
+    "type": "object",
+    "properties": {
+        "describedBy": {
+            "description": "The URL reference to the schema.",
+            "type": "string",
+            "pattern" : "^(http|https)://schema.(.*?)humancellatlas.org/module/project/(([0-9]{1,}.[0-9]{1,}.[0-9]{1,})|([a-zA-Z]*?))/contact"
+        },
+        "schema_version": {
+            "description": "The version number of the schema in major.minor.patch format.",
+            "type": "string",
+            "pattern": "^[0-9]+\\.[0-9]+\\.[0-9]+$",
+            "example": "4.6.1"
+        },
+        "name": {
+            "description": "A field describing which HCA Bionetwork the project is a part of (e.g. Kidney). Enter ‘None’ if the project is not part of an official HCA Bionetwork.",
```

```suggestion
            "description": "Name of the HCA Bionetwork the associated project is a part of (e.g. Kidney).",
```

I wouldn't use "A field ..." for the description, there's nothing in the guidelines but it sounds a little weird to me! Also the `Enter ‘None’ if the project is not part of an official HCA Bionetwork.` seems more fit in the guidelines :)